### ISSUE TITLE
Inconsistent authentihash generated

### ISSUE THREAD

**Issue by gdesmar** [lief-project/LIEF#932]
**Describe the bug**
Requesting the authentihash multiple time returns different results for the same file.

**To Reproduce**
After installing LIEF 0.12.x, the following returns "06044cdc4296fe1462fbe86b8079097a2752fc586556b0c61f84d121b589e7ef"
```bash
python -c "import lief; print(lief.parse('2ef76414c6fb9931a6a004694ce69ca3178a99d0b77b9b471f8e4d4222b9a90f').authentihash_sha256.hex())"
```
Once 0.13.0 or 0.13.1 is installed, the value keeps changing on every execution.

**Expected behavior**
VirusTotal agrees that it should be what it was in 0.12.x.

**Environment (please complete the following information):**
 - System and Version : Debian 11 Bullseye
 - Target format : PE

**Additional context**
A second sample, 202260c2ee4906215108dcbc56306fc688583a9f32c880aa833a0fdf32780067 is also provided in the zip file in case it would help narrow the issue. Others sample can be uploaded if needed. Both files are classified as **malicious** on VirusTotal. The password for the [authentihash_samples.zip](https://github.com/lief-project/LIEF/files/11761980/authentihash_samples.zip) file is `liefsample`.

**Comment by romainthomas** [comment]
Indeed there is a regression that has been implicitly fixed here: [Bump LIEF to C++17 ](https://github.com/lief-project/LIEF/commit/2f2e5bf7fe722020db78e66de1ea06b0da538251#diff-9badf498f121605fbe7ebe932c1eaef1ccbf4e87d32e3e41ada7915ea0e6ebd6R54-R61). 

This fix will be back-ported for the `0.13.2` release

**Comment by romainthomas** [comment]
Fixed with [0.13.2](https://github.com/lief-project/LIEF/releases/tag/0.13.2)
